app/helpers/application_helper/title.rb

```diff
@@ -1,119 +1,122 @@
 module ApplicationHelper
   module Title
     def productized_title(title)
-      product_title + title
+      if title.present?
+        "%{product}: %{title}" % {:product => product_title, :title => title}
```

Comment:
Do we want to i18n this string (the `:` basically), @mzazrivec ?

Reply:
I'd say no for now. We can fix this later, should that be a problem in some locales.